modules/navier_stokes/test/tests/finite_volume/ins/channel-flow/linear-segregated/2d/2d-velocity-pressure.i

```diff
@@ -0,0 +1,167 @@
+    type = MooseLinearVariableFVReal
+    solver_sys = v_system
+    initial_condition = 0.0
+  []
+  [pressure]
+    type = MooseLinearVariableFVReal
+    solver_sys = pressure_system
+    initial_condition = 0.2
+  []
+[]
+
+[LinearFVKernels]
+  [u_advection_stress]
+    type = LinearWCNSFVMomentumFlux
+    variable = vel_x
+    advected_interp_method = ${advected_interp_method}
+    mu = ${mu}
+    u = vel_x
+    v = vel_y
+    momentum_component = 'x'
+    rhie_chow_user_object = 'rc'
+    use_nonorthogonal_correction = false
+  []
+  [v_advection_stress]
+    type = LinearWCNSFVMomentumFlux
+    variable = vel_y
+    advected_interp_method = ${advected_interp_method}
+    mu = ${mu}
+    u = vel_x
+    v = vel_y
+    momentum_component = 'y'
+    rhie_chow_user_object = 'rc'
+    use_nonorthogonal_correction = false
+  []
+  [u_pressure]
+    type = LinearFVMomentumPressure
+    variable = vel_x
+    pressure = pressure
+    momentum_component = 'x'
+  []
+  [v_pressure]
+    type = LinearFVMomentumPressure
+    variable = vel_y
+    pressure = pressure
+    momentum_component = 'y'
+  []
+  [p_diffusion]
+    type = LinearFVAnisotropicDiffusion
+    variable = pressure
+    diffusion_tensor = Ainv
+    use_nonorthogonal_correction = false
+  []
+  [HbyA_divergence]
+    type = LinearFVDivergence
+    variable = pressure
+    face_flux = HbyA
+    force_boundary_execution = true
+  []
+[]
+
+[LinearFVBCs]
+  [inlet-u]
+    type = LinearFVAdvectionDiffusionFunctorDirichletBC
+    boundary = 'left'
+    variable = vel_x
+    functor = '1.1'
+  []
+  [inlet-v]
+    type = LinearFVAdvectionDiffusionFunctorDirichletBC
+    boundary = 'left'
+    variable = vel_y
+    functor = '0.0'
+  []
+  [walls-u]
+    type = LinearFVAdvectionDiffusionFunctorDirichletBC
+    boundary = 'top bottom'
+    variable = vel_x
+    functor = 0.0
+  []
+  [walls-v]
+    type = LinearFVAdvectionDiffusionFunctorDirichletBC
+    boundary = 'top bottom'
+    variable = vel_y
+    functor = 0.0
+  []
+  [outlet_p]
+    type = LinearFVAdvectionDiffusionFunctorDirichletBC
+    boundary = 'right'
+    variable = pressure
+    functor = 1.4
+  []
+  [outlet_u]
+    type = LinearFVAdvectionDiffusionOutflowBC
+    variable = vel_x
+    use_two_term_expansion = false
+    boundary = right
+  []
+  [outlet_v]
+    type = LinearFVAdvectionDiffusionOutflowBC
+    variable = vel_y
+    use_two_term_expansion = false
+    boundary = right
+  []
+[]
+
+[Executioner]
+  type = SIMPLE
+  momentum_l_abs_tol = 1e-10
+  pressure_l_abs_tol = 1e-10
+  momentum_l_tol = 0
+  pressure_l_tol = 0
+  rhie_chow_user_object = 'rc'
+  momentum_systems = 'u_system v_system'
+  pressure_system = 'pressure_system'
+  momentum_equation_relaxation = 0.8
+  pressure_variable_relaxation = 0.3
+  num_iterations = 50
+  pressure_absolute_tolerance = 1e-10
+  momentum_absolute_tolerance = 1e-10
+  momentum_petsc_options_iname = '-pc_type -pc_hypre_type'
+  momentum_petsc_options_value = 'hypre boomeramg'
```

Comment:
How does this do on an advection dominated problem?

Reply:
I onl ytested it upt to Re=100 with inflow-outfflow problems. But not bad actually. Still solves the momentum equation in 3-10 iterations. I didn't examine the mesh size dependence though.